### PR TITLE
Added onclick action parameter for custom grid btn

### PR DESCRIPTION
**Pull request by Kkeller83**
Added onclick action parameter for custom grid buttons in order fire javascript functions.

There appears to be more changes than I actually implemented, since I used :retab!4 in vim to have unified indentation in the file. 

Actual lines changed: 

586                 onclick=None, (added)

864                                                 if btn.onclick: (added)
865                                                                 btn.url=None (added)

 873                                                                 _onclick=btn.onclick, (added)

Thats it.

### CODE REVIEW

**Comment by jpsteil** [comment]
Kevin - Looks like your line re-formatter really did a number on this file.  Any chance you can cancel this and submit a patch without the reformatted lines?

**Comment by Kkeller83** [comment]
Let me try again :)
